### ISSUE TITLE
delete from mapping throw thread panic error at Solang

### ISSUE THREAD

**Issue by onahornyak** [hyperledger-labs/solang#861]
Hello,

This contact work for solidity, but for solang throw "thread 'main' panicked" at `delete example[id];`

```
contract a {

    struct data_struct  {
          address addr1;
	  address addr2;
    }

    mapping(uint => data_struct) example;

    function deltest() external {
        uint id = 1;
        delete example[id]; // this line throw "thread 'main' panicked"
    }

}
```

thread 'main' panicked at 'Found ArrayType(ArrayType { array_type: Type { address: 0x55a57c6b1390, llvm_type: "[32 x i8]" } }) but expected the IntType variant', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/inkwell-0.1.0-beta.4/src/types/enums.rs:390:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

Pleas fix this issue,

Thank you.

**Comment by LucasSte** [comment]
Thanks for reporting this. I'll have a look at the problem, and I'll get back to you.

**Comment by LucasSte** [comment]
I've found a fix already: #865.

**Comment by onahornyak** [comment]
Perfect. I will wait for this fix
